react-accordion-component/my-app/src/Accordion.tsx

```diff
@@ -0,0 +1,40 @@
+import { useState } from 'react';
+
+type Items = {
+  id: number;
+  title: string;
+  content: string;
+};
+
+export function Accordion({ arr }: { arr: Items[] }) {
+  const [id, setId] = useState(0);
+
+  const item = arr.map((i) => (
+    <RenderTopic item={i} key={i.id} id={id === i.id} setId={setId} />
+  ));
+  return item;
+}
+
+type List = {
+  item: Items;
+  id: boolean;
+  setId: (num: number) => void;
```

Comment:
`id` is a strange name for a boolean. I don't understand what it's for just based on its name.

And why does `setId` take a number when `id` is a boolean?